src/styles/index.scss

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  display: flex;
+  border-color: #000;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+}
+
+.stopwatch__center {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 5vmin;
+  height: 5vmin;
+  background-color: #f6a603;
+  border-radius: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 10;
+}
+
+.stopwatch__hand {
+  position: absolute;
+  bottom: 50%;
+  left: 50%;
+  transform-origin: bottom center;
+  transform: translateX(-50%);
+}
+
+.stopwatch__hand--minutes {
+  width: 3vmin;
+  height: 20vmin;
+  background-color: #0700ff;
+  animation: rotate-minutes 3600s steps(60) infinite;
+}
+
+.stopwatch__hand--seconds {
+  width: 1.5vmin;
+  height: 38vmin;
+  background-color: #2c8000;
+  animation: rotate-seconds 60s linear infinite;
+}
+
+@keyframes rotate-minutes {
+  from {
+    transform: translateX(-50%) rotate(0deg);
+  }
+  to {
+    transform: translateX(-50%) rotate(360deg);
+  }
+}
+
+@keyframes rotate-seconds {
+  from {
+    transform: translateX(-50%) rotate(0deg);
+  }
+  to {
+    transform: translateX(-50%) rotate(360deg);
+  }
```

Comment:
looks like two the same keyframes, feel free to remove one of them

Reply:
there are two different keyframe for both watch-hands. if i remove one of them, one hand won't be works. or i am wrong? i didn't find how i can optimize these frame more(

Reply:
Not fixed, use only one keyframes for two arrows

Reply:
Feel free to ask for help in the chat to fix it